packages/cli/package.json

```diff
@@ -7,17 +7,15 @@
   },
   "bugs": "https://github.com/stoplightio/prism/issues",
   "dependencies": {
-    "@stoplight/http-spec": "^2.2.3",
+    "@stoplight/http-spec": "2.2.3",
```

Comment:
Are we pegging to a specific version intentionally? we normally add `^` but this removes one.

Reply:
Fixed. I do not know how this happened.